controllers/eventController.js

```diff
@@ -118,26 +130,12 @@ exports.generate_pdf = function (req, res) {
 }
 
 exports.generate_daily_events = function (req, res) {
-  function snapshotToArray (snapshot) {
-    var returnArr = []
-
-    snapshot.forEach(function (childSnapshot) {
-      var item = childSnapshot.val()
-      returnArr.push(item)
-    })
-
-    return returnArr
-  }
   var date = req.query.date
   var filename = path.join(__dirname, `events-${date}.pdf`)
   console.log(date)
   var eventsRef = admin.database().ref('to-be-held/' + date)
   var eventRef = admin.database().ref()
-  // console.log(eventsRef);
   eventsRef.once('value', function (snapshot) {
-    // if(snapshot.val()===null) {
-    //   console.log("No events Booked for this day")
-    // }
     var html
     var eventID = snapshotToArray(snapshot)
```

Comment:
You could check the length of the array and respond if it is 0 here itself instead of using the counter `i`.